### ISSUE TITLE
Acquire another freshdesk agent slot for Pris, and document how 

### ISSUE THREAD

**Issue by consideRatio** [2i2c-org/team-compass#668]
As we onboard Pris, we require another agent slot. I've worked around this for now by putting @sgibson91 as an occational agent for now.

### Action point

- [ ] Acquire a freshdesk slot for Pris
- [ ] Summarize process of acquring a new slot in this github issue
- [ ] Document process of acquiring a new slot in https://compass.2i2c.org/en/latest/community/freshdesk.html

### Related

- https://github.com/2i2c-org/team-compass/issues/608

**Comment by damianavila** [comment]
I think there is no need, for now, to actually buy a new seat because Camille can be turned into an occasional agent and use day passes (instead of a full seat since she has not logged in in more than 2 months) and I can make @sgibson91 a full agent again.

I have proceeded with that change, @sgibson91 is now a full agent again.

If we need to buy a new seat, that will be a seat for Camille, so I have edited the title and the top message of this issue to account for that.

**Comment by damianavila** [comment]
For the record, at the same time, I have also bumped @pnasrat to the Admin role (like any other eng doing support).
We may want to revisit the permission model later down the road, but for now, let's put everyone on the Eng team at the same level of permissions.

**Comment by choldgraf** [comment]
This makes sense to me - so this issue is just now documenting how to add agent slots?

**Comment by damianavila** [comment]
> so this issue is just now documenting how to add agent slots?

Indeed, I have updated the title and the first message of the issue to account for that.
